CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing Guidelines
+
+Contributions are welcome via [GitHub Pull Requests](https://docs.github.com/articles/about-pull-requests). This document outlines the process to help get your contribution accepted.
```

Comment:
Contributions are welcome via [GitHub Pull Requests](https://docs.github.com/articles/about-pull-requests) ("PR"). This document outlines the process to help get your contribution accepted.